web/libs/ui/src/lib/button/button.module.scss

```diff
@@ -0,0 +1,328 @@
+.base {
+  --background-image: none;
+  --background-color: var(--color-primary-surface);
+  --border-color: var(--color-primary-border-bold);
+  --border-color-hover: var(--border-color);
+  --text-color: var(--color-primary-surface-content);
+  --focus-outline: var(--color-primary-focus-outline);
+  --background-color-hover: var(--color-primary-surface-hover);
+  --background-color-active: var(--color-primary-surface-active);
+  --wait-color-value: var(--color-primary-emphasis-subtle);
+  --wait-color-value-outline: var(--color-primary-emphasis);
+  --wait-color-opacity: 10%;
+  --wait-color: color-mix(in lab, var(--wait-color-value) var(--wait-color-opacity), transparent);
+  --focus-shadow: 0 0 0 4px var(--focus-outline);
+  --emboss-shadow: inset 0 1px 0 rgb(255 255 255 / 10%), inset 0 -1px 0 rgb(0 0 0 / 10%);
+  --emboss-shadow-active: inset 0 -1px 0 rgb(255 255 255 / 5%), inset 0 1px 0 rgb(0 0 0 / 10%);
```

Comment:
I don't think any of these are dark mode compatible. 

- all of these shadows use `0 0 0` or `255 255 255` directly instead of a raw semantic token that adheres to color-scheme.

Can you merge in latest develop to this branch so we can check it all with the dark mode toggle that was added to storybook?